nativeRun.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+printf "\n\nStarting local Jekyll build and serving site \n\n"
+
+bundle install
+bundle exec jekyll clean
+bundle exec jekyll serve --trace --incremental $1
```

Comment:
This doesn't seem related to the PR, should probably revert changes to this file